tests/dtypes_test.py

```diff
@@ -0,0 +1,20 @@
+from commont import *
+
+
+def test_dtype(ds_local):
+  ds = ds_local
+  assert ds.dtype(ds.x) == np.float64
+  assert ds.dtype(ds.y) == np.float64
+  assert ds.dtype(ds.m) == np.float64
+  assert ds.dtype(ds.nm) == np.float64
+  assert ds.dtype(ds.mi) == np.int64
+  assert ds.dtype('ints') == np.int64
+  assert ds.dtype('name') == 'S25'
+  assert ds.dtype('obj') == np.object
+
+def test_dtypes(ds_local):
+  ds = ds_local
+  all_dtypes = np.array([np.float64, np.float64, np.float64, np.float64, np.int64, np.int64, 'S25', np.object])
+  np.testing.assert_array_equal(ds.dtypes(columns=None), all_dtypes)
```

Comment:
On writing tests, don't use np.array for non-numerical stuff, it's not idiomatic Python, better/simpler would be:
```
all_dtypes = [np.float64, ....]
ds.dtypes(None) == all_dtypes
```
Less code, easier to read, and the use of numpy only confuses.

Reply:
This has nothing to do with numpy, so I recommend just use assert (otherwise it implies some numerical equality, which is not tested here)
assert ds.dtypes(columns=None).values == all_dtypes